### PR TITLE
Read receipts

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: src/controllers/organisms/RoomView.js [hunk 1]
```diff
@@ -529,7 +542,7 @@ module.exports = {
             }
 
             ret.unshift(
-                <li key={mxEv.getId()}><EventTile mxEvent={mxEv} continuation={continuation} last={last}/></li>
+                <li key={mxEv.getId()} ref={this._collectEventNode.bind(this, mxEv.getId())}><EventTile mxEvent={mxEv} continuation={continuation} last={last}/></li>
```
Comment (Kegsay) [inline src/controllers/organisms/RoomView.js]: This looks like the same stuff we're doing in React SDK?

File: src/skins/vector/views/molecules/EventTile.js [hunk 2]
```diff
@@ -17,15 +17,20 @@ limitations under the License.
...
 var sdk = require('matrix-react-sdk')
+var MatrixClientPeg = require('matrix-react-sdk/lib/MatrixClientPeg')
 
 var EventTileController = require('matrix-react-sdk/lib/controllers/molecules/EventTile')
 var ContextualMenu = require('../../../../ContextualMenu');
 
 var TextForEvent = require('matrix-react-sdk/lib/TextForEvent');
 
+var Velociraptor = require('../../../../Velociraptor');
+require('../../../../VelocityBounce');
```
Comment (Kegsay) [inline src/skins/vector/views/molecules/EventTile.js]: Please comment what the side-effects are.

**Comment by Kegsay** [comment]
LGTM though I can't really say much about the velocity stuff. The PR is too large for my poor brain to grok it fully. From what I can grok, it looks good though :D

File: src/controllers/organisms/RoomView.js (hunk 1 above)
Comment (dbkr) [inline src/controllers/organisms/RoomView.js]: Yep, trying to merge the same changes over to the duplicated controller in react sdk :(

**Pull request merged**